Tests/SentryTests/SentryNSURLRequestTests.swift

```diff
@@ -11,10 +12,21 @@ class SentryNSURLRequestTests: XCTestCase {
     
     func testRequestWithEnvelopeEndpoint() {
         let request = try! SentryNSURLRequest(envelopeRequestWith: SentryNSURLRequestTests.dsn(), andData: Data())
-        XCTAssertTrue(request.url!.absoluteString.hasSuffix("/envelope/"))
+        expect(request.url!.absoluteString).to(endWith("/envelope/"))
     }
+    
     func testRequestWithStoreEndpoint() {
         let request = try! SentryNSURLRequest(storeRequestWith: SentryNSURLRequestTests.dsn(), andData: Data())
-        XCTAssertTrue(request.url!.absoluteString.hasSuffix("/store/"))
+        expect(request.url!.absoluteString).to(endWith("/store/"))
+    }
+    
+    func testRequestWithEnvelopeEndpoint_hasUserAgentWithSdkNameAndVersion() {
+        let request = try! SentryNSURLRequest(envelopeRequestWith: SentryNSURLRequestTests.dsn(), andData: Data())
+        expect(request.allHTTPHeaderFields?["User-Agent"]).to(beginWith(String(format: "%@/%@", SentryMeta.sdkName, SentryMeta.versionString)))
```

Comment:
l: Im not sure about using `beginWith` I would rather having a equality comparison, so we ensure no trash is added to the end of the `user-agent`.

```suggestion
        expect(request.allHTTPHeaderFields?["User-Agent"]) == "\(SentryMeta.sdkName)/\(SentryMeta.versionString)"
```